[keymap] Added abnt2 format to dz60 (#7340)
qmk/qmk_firmware: keyboards/dz60/keymaps/iso_br_abnt2/readme.md

> # ISO-BR abnt2 layout

Reviewer: ```suggestion
# BR ABNT2 layout
```

> This is an ISO-BR abnt2 keymap for the DZ60 in a standard ISO layout.

Reviewer: ```suggestion
This is a Brazilian ABNT2 keymap for the DZ60 in a standard layout.
```

The top-left sends Escape normally, but becomes a quote key when
tapped whilst either Fn or GUI are held down.

## Layout Image

> ![Layout Image](https://upload.wikimedia.org/wikipedia/commons/thumb/2/2c/KB_Portuguese.svg/1200px-KB_Portuguese.svg.png)

Reviewer: This layout image doesn't match the physical layout (no split on the right Shift).

> ![Layout Image](https://upload.wikimedia.org/wikipedia/commons/thumb/2/2c/KB_Portuguese.svg/1200px-KB_Portuguese.svg.png)

Author: ```suggestion
![Layout Image](http://s2.glbimg.com/hlQMZ_E87B3DB_96rVuO_pY2Yy8=/695x0/s.glbimg.com/po/tt2/f/original/2016/06/21/900px-kb_portuguese_brazilsvg_.png)
```
